x11-misc/xkeyboard-config/xkeyboard-config-9999.ebuild

```
@@ -28,14 +28,32 @@ BDEPEND="
 	dev-libs/libxslt
 	sys-devel/gettext
 	test? (
+		x11-apps/xkbcomp
+		x11-libs/libxkbcommon
 		$(python_gen_any_dep '
+			dev-python/pycountry[${PYTHON_USEDEP}]
+			dev-python/pytest-xdist[${PYTHON_USEDEP}]
 			dev-python/pytest[${PYTHON_USEDEP}]
 		')
 	)
 "
 
+python_check_deps() {
+	use test || return 0
+	python_has_version "dev-python/pycountry[${PYTHON_USEDEP}]"
+	python_has_version "dev-python/pytest-xdist[${PYTHON_USEDEP}]"
+	python_has_version "dev-python/pytest[${PYTHON_USEDEP}]"
+}
+
 pkg_setup() {
-	python-any-r1_pkg_setup
+	use test && python-any-r1_pkg_setup
```

`PYTHON_DEPS` is listed in BDEPEND unconditionally, so I think the double `use test &&` here in both `pkg_setup` _and_ `python_check_deps` is wrong?

I think you want to drop `use test &&` here:
```suggestion
	python-any-r1_pkg_setup
```